dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.org.eclipse.microprofile.rest.client-1.4.feature

```diff
@@ -0,0 +1,11 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.org.eclipse.microprofile.rest.client-1.4
+singleton=true
+-features=com.ibm.websphere.appserver.javax.cdi-2.0, \
+ com.ibm.websphere.appserver.javax.annotation-1.3, \
+ com.ibm.websphere.appserver.javax.jaxrs-2.1, \
+ com.ibm.websphere.appserver.org.eclipse.microprofile.config-1.3; ibm.tolerates:=1.4
```

Comment:
should use config 1.4 by default since that is part of MP 3.3

Reply:
Done